package/gluon-core/luasrc/lib/gluon/upgrade/190-preserve-wireless-channels

```diff
@@ -1,23 +0,0 @@
-#!/usr/bin/lua
-
-local wireless = require 'gluon.wireless'
-local uci = require('simple-uci').cursor()
-
-local preserve_channels = wireless.preserve_channels(uci)
-
--- Migrate preserve channels from pre-2022.01
-local core_wireless = uci:get_first('gluon-core', 'wireless')
-if core_wireless ~= nil then
-	local preserve_legacy = uci:get_bool('gluon-core', core_wireless, 'preserve_channels')
-	if preserve_legacy then
-		preserve_channels = true
-	end
-
-	uci:delete('gluon-core', core_wireless)
-	uci:save('gluon-core')
-end
-
-uci:section('gluon', 'wireless', 'wireless', {
-	preserve_channels = preserve_channels or false,
-})
-uci:save('gluon')
```

Comment:
Hmm, maybe we want to keep the initial setup of `gluon.wireless.preserve_channels` and only delete lines 8..18?

Reply:
Fine for me as well, so that we keep setting the default value if it does not exist.
I just pushed an update to this.